src/Http/Controllers/API/FilterController.php

```diff
@@ -10,22 +11,44 @@
 
 class FilterController extends Controller
 {
+    protected $filterBuilderPath;
+
+    protected $activityLogModel;
+
+    protected $userModel;
+
+    protected $userSearch;
+
+    protected $userSearchRelationship;
+
+    protected $userSearchTerm;
+
+    protected $defaultFilterPagination;
+
     public function __construct()
     {
+        $this->filterBuilderPath = config('activity-log.filter_builder_path', 'App\Support\Filters\FilterBuilder');
```

Comment:
can we remove default value for this. we already add default value in config file